carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -1215,8 +1217,8 @@ void WMBroadcaster::setIntersectionCoordCorrection(const std::vector<int>& inter
 
   for (auto i = 0; i != intersection_correction.size(); i+2)
   {
-    sim_.intersection_coord_correction_[intersection_ids_for_correction[i/2]].first =  intersection_correction[i]; //x
-    sim_.intersection_coord_correction_[intersection_ids_for_correction[i/2]].second = intersection_correction[i + 1]; //y
+    sim_.intersection_coord_correction_[(int)intersection_ids_for_correction[i/2]].first =  intersection_correction[i]; //x
```

Comment:
is int the most accurate cast here? Or int64_t since you were using that above

Reply:
I think actually the right cast is `uint16_t`[ since this is the format which SPAT/MAP sends to CARMA.](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/carma_wm/include/carma_wm/SignalizedIntersectionManager.h#L145) I specified the range in yaml to avid overlow